supertokens_python/recipe/thirdpartypasswordless/utils.py

```diff
@@ -117,14 +123,18 @@ def __init__(self,
                  contact_config: ContactConfig,
                  flow_type: Literal['USER_INPUT_CODE', 'MAGIC_LINK', 'USER_INPUT_CODE_AND_MAGIC_LINK'],
                  get_link_domain_and_path: Callable[[PhoneOrEmailInput, Dict[str, Any]], Awaitable[str]],
-                 get_custom_user_input_code: Union[Callable[[
-                     Dict[str, Any]], Awaitable[str]], None] = None):
+                 get_email_delivery_config: Callable[
+                     [RecipeInterface], EmailDeliveryConfigWithService[TypeThirdPartyPasswordlessEmailDeliveryInput]
+                 ],
+                 get_custom_user_input_code: Union[Callable[[Dict[str, Any]], Awaitable[str]], None] = None
+                 ):
```

Comment:
Need to mark old functions for sending email as deprectaed

Reply:
Updated `tppless.utils.InputEmailVerificationConfig`